Add support for handling multiple identify results (#106)
Esri/data-collection-dotnet: RELEASE.md

    * Switches to vector icons where possible.
* Moves the .NET Core WPF project to a separate folder to prevent possible build issues.
* Updates the code called at sign out to revoke credentials from the server, when possible, and delete them locally.

> * Adds support for identifying mulitple items on the map simultaneously. A unique interface gives you enough context to choose a result, with context display configurable via the web map.

Reviewer: I'm not really sure what this means, can you explain it? Perhaps we remove it altogether?

> A unique interface gives you enough context to choose a result, with context display configurable via the web map.

Maybe you meant something like,

"Users can interrogate the list to display a feature's full details."

```suggestion
* Adds support for identifying multiple features on the map and displaying them in a list.
```

Author: I was referencing the subtitle feature indirectly, maybe it makes sense to call that out more explicitly? It is a somewhat unique pattern that I haven't seen elsewhere.

    * Switches to vector icons where possible.
* Moves the .NET Core WPF project to a separate folder to prevent possible build issues.
* Updates the code called at sign out to revoke credentials from the server, when possible, and delete them locally.
* Adds support for identifying mulitple items on the map simultaneously and displaying them in a list.

>     * An arcade expression defined per layer is used to create subtitles that help you differentiate identified features.

Reviewer: I believe Arcade is a proper noun and should be capitalized. Also, this should be optional, no?

```suggestion
    * An Arcade expression defined per layer can be used to create subtitles that help you differentiate identified features.
```
